Components/Snackbar/index.css

```diff
@@ -0,0 +1,18 @@
+.kash-snackbar {
+  padding: calc(0.6 * var(--padding));
+  color: var(--white);
+  gap: 1.5rem;
+  border-radius: 0.3rem;
+  position: fixed;
+  right: 1rem;
+  top: 5rem;
+  display: none;
+}
+
+.kash-snackbar > div > .kash-btn-text {
+  margin: 0;
+}
```

Comment:
no need to be too much spacific. I think only class name will do your thing. Try that..

Reply:
Sure